mysql-test/suite/innodb/t/percona_session_temp_tablespaces_encrypt.test

```diff
@@ -0,0 +1,109 @@
+let $MYSQLD_DATADIR=`SELECT @@datadir`;
+
+# Using on disk tablespace for intrinsic optimiser tables
+SELECT @@innodb_temp_tablespace_encrypt;
+SET GLOBAL big_tables=ON;
+
```

Comment:
Later: surround the testcase with count_sessions.inc / wait_until_count_sessions.inc